File: Counting/1742.Maximum Number of Balls in a Box/Solution.py
class Solution:
    def countBalls(self, lowLimit: int, highLimit: int) -> int:
        cnt = [0] * 46
        for i in range(lowLimit, highLimit + 1):
            num, index = i, 0
            while num :
                index += num % 10
                num //= 10
            cnt[index] += 1
        return max(cnt)

File: Counting/1742.Maximum Number of Balls in a Box/Solution.java
class Solution {
    public int countBalls(int lowLimit, int highLimit) {
        int[] cnt = new int[46];
        for (int i = lowLimit; i < highLimit + 1; i++) {
            int num = i, index = 0;
            while (num > 0) {
                index += num % 10;
                num /= 10;
            }
            cnt[index]++;
        }
        int ans = 0;
        for (int v : cnt) {
            ans = Math.max(ans, v);
        }
        return ans;
    }
}